kbfsgit/start.go

```diff
@@ -60,7 +61,9 @@ func Start(ctx context.Context, options StartOptions,
 	}
 	defer config.Shutdown(ctx)
 
-	r, err := newRunner(ctx, config, options.Repo, input, output)
+	r, err := newRunner(
+		ctx, config, options.Remote, options.Repo, options.GitDir,
```

Comment:
Do you think it'd be clearer if we name stuff as "OnDisk" and "BareRepo" rather than "Remote"?

Reply:
I'm not sure, I was using the names from the doc to keep things consistent: https://git-scm.com/docs/git-remote-helpers.  For now I'll leave it as is, and add comments explaining what these names mean to us.